sdk/daml-assistant/integration-tests/src/DA/Daml/Assistant/IntegrationTests.hs

```diff
@@ -126,7 +126,7 @@ damlStart tmpDir disableUpgradeValidation = do
             , "    output-directory: ui/java"
             , "build-options:"
             , "- --target=2.1"
-            ] ++ [ "- --warn-bad-interface-instances=yes" |  disableUpgradeValidation ]
+            ] ++ [ "- -Wupgrade-interfaces" |  disableUpgradeValidation ]
```

Comment:
Given its `disableUpgradeValidation`, should this not fully disable the warnings?
(as per review on #20207)

Reply:
done